ui/models/job.js

```diff
@@ -118,35 +120,46 @@ export default class JobModel {
     try {
       notify(`Attempting to retrigger ${jobTerm} via actions.json`, 'info');
 
+      const uniquePerPushJobs = {};
+
       /* eslint-disable no-await-in-loop */
       for (const id of jobIds) {
         const job = await JobModel.get(repoName, id);
+
+        if (!uniquePerPushJobs[job.push_id]) {
+          uniquePerPushJobs[job.push_id] = [];
+        }
+        uniquePerPushJobs[job.push_id].push(job.job_type_name);
+      }
+
+      /* eslint-disable no-await-in-loop */
```

Comment:
Could you rework this so we don't need this disable line? See:
https://eslint.org/docs/rules/no-await-in-loop

Reply:
Done.